packages/mdc-fab/_mixins.scss

```diff
@@ -40,7 +40,10 @@
   width: 56px;
   height: 56px;
   padding: 0;
-  transition: box-shadow 280ms $mdc-animation-standard-curve-timing-function;
+  transition:
+    box-shadow 280ms $mdc-animation-standard-curve-timing-function,
+    opacity 15ms linear 30ms,
+    mdc-animation-enter(transform, 270ms);
```

Comment:
is 270ms here the 180ms + 90ms from below? If so, maybe reference some private variables directly

Reply:
The specs don't necessarily directly call this out (which is why this didn't occur to me), but the idea makes sense to me since the two motions do line up.